### DISCUSSION TITLE
Use bullMQ with express.js

### DISCUSSION THREAD

**Discussion by tomeof** [taskforcesh/bullmq#1132] [Q&A]
I am trying to use bullMQ flows with express.js based API. When level2 jobs finish, level1 job gets the results from the children, does some extra processing and returns a result (a single number). How can i send the final result from level1 queue inside an express.js request handler?
```javascript
const express = require('express');
const { FlowProducer } = require('bullmq');

const app = express();

const flowProducer = new FlowProducer({
  connection: {
    host: "localhost",
    port: 6379
  }
});

app.get('/', async (req, res) => {

  let children = [];
  for (var i = 0; i < foo.length; i++) {
    children.push({
      name: 'myJob',
      data: { bar: foo[i] },
      queueName: 'level2'
    });
  }

  const flow = await flowProducer.add({
    name: 'myFlow',
    queueName: 'level1',
    children: children
  });

  res.send('finished');
})

app.listen(3000);
```

**Comment by reginsmol** [comment]
I wouldn't suggest relying on finishing jobs to return back a response to the client, since there's no guarantee that the job will be finished in a reasonable amount of time for a response.
